server/src/main/java/org/elasticsearch/rest/BaseRestHandler.java

```diff
@@ -270,4 +272,10 @@ protected Set<String> responseParams(RestApiVersion restApiVersion) {
         return responseParams();
     }
 
+    protected void setErrorTraceTransportHeader(RestRequest r, ThreadContext c) {
```

Comment:
this does not access any state of the handler, does it? Is it supposed to be a static utility method then?

Reply:
Done!